docs/howto/upgrade-cluster/aws.md

```diff
@@ -216,11 +271,17 @@ cluster is unused or that the maintenance is communicated ahead of time.
    ```
    ````
 
-6. *Repeat steps 3 and 5 if needed*
+### 7. Repeat steps 4 and 6 if needed
 
-   If you upgrade k8s multiple minor versions, repeat step 3 and 5, where you
+   If you upgrade k8s multiple minor versions, repeat step 4 and 6, where you
    increment it one minor version at the time.
 
```

Comment:
I think this step stops making sense after the change to let a step upgrade the k8s control plane multiple times in a row, so I figure this should be removed!